examples/piecestore/rpc/client/utils/utils.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package utils
+
+import (
+	"crypto/md5"
+	"fmt"
+	"io"
+	"os"
+)
+
+// DetermineID -- Get the id for a section of data
+func DetermineID(f *os.File, offset int64, length int64) (string, error) {
+	h := md5.New()
```

Comment:
did you see my comment about md5 from earlier?

Reply:
you gotta figure out what's happening with previous github comments my dude

Reply:
👍